templates/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "Saleor demo shop",
+  "description": "Saleor is best storefront engine evar",
+  "short_name": "Saleor Demo",
+  "icons": [
+    {
+      "src": "/static/assets/favicons/android-chrome-36x36.png",
```

Comment:
Those paths may be different depending on the environment. I think this whole file should be generated by Django.